CrossPlatform/SynCrossPlatformREST.pas

```diff
@@ -2974,8 +2974,9 @@ procedure TSQLRestClientURI.SetAsynch(var Call: TSQLRestURIParams;
             onError(self);
       if fAsynchCount>0 then
         dec(fAsynchCount);
-      if fAsynchCount=0 then
-        CallAsynchText; // send any pending asynchronous task
+      if fAsynchCount=0 then 
+        if Assigned(self) then

```

Comment:
if self is void, then fAsynchCount access in the above line would fail - so what is it about?

Reply:
Good point. The main problem is that the fAsynchCount works and don't want to fail but at the same time CallAsynchText fails. I need to find some alternative to SMS... This is just hot-fix, I was not able to find the root of problem (and don't have much motivation for this - only root of DWScript is open source but whole JS part is not usable without commercial RTL).